app/Example.java

```diff
@@ -0,0 +1,50 @@
+import android.view.View;
+
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.Channel;
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.Message;
+import io.ably.lib.types.Param;
+
+public class Example {
+
+	private static final String API_KEY = "INSERT-YOUR-API-KEY-HERE";
+
+	public void publishAndSubcscribe(Button buttonPublish, Button buttonHistory) {
+		AblyRealtime realtime = new AblyRealtime(API_KEY);
```

Comment:
There is a very little difference between AblyRest and AblyRealtime in terms of code usage, you just need adjust imports for your Channel class and change AblyRealtime to AblyRest. The rest in this tutorial works the same. Maybe instead of making two separate tutorials, we can address this in docs?